common/config/doc_generator/main.go

```diff
@@ -16,4 +17,8 @@ func main() {
 
 	err = config.DocumentConfig(ejector.DefaultRootEjectorConfig, configDocsDir, true)
 	enforce.NilError(err, "failed to generate docs for the ejector config")
+
+	// requireDocs is false because nested configs (blobstore, chunkstore, etc.) may not have complete GoDoc comments
+	err = config.DocumentConfig(encoder.DefaultRootEncoderConfig, configDocsDir, false)
+	enforce.NilError(err, "failed to generate docs for the encoder config")
```

Comment:
We'll want to circle back at some point in time to fix this, but ok to do that later and in a separate PR.